src/client/Microsoft.Identity.Client/Internal/Logger/LoggerHelper.cs

```diff
@@ -110,6 +113,8 @@ internal static string GetPiiScrubbedExceptionDetails(Exception ex)
                 {
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "HTTP StatusCode {0}", msalServiceException.StatusCode));
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "CorrelationId {0}", msalServiceException.CorrelationId));
+                    var aadError = JsonHelper.GetExistingOrEmptyString(JsonHelper.ParseIntoJsonObject(msalServiceException.ResponseBody), OAuth2ResponseBaseClaim.ErrorDescription).Split(':').FirstOrDefault();
```

Comment:
maybe you can move this into a `ParseAadErrorCode` function?

Reply:
also string interpolation would help with readability

Reply:
removed parsing